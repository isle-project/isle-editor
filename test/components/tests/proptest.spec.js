// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import PropTest from '@isle-project/components/tests/proptest';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<PropTest />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <PropTest
			data={heartdisease}
			variable="Complications"
			success={1}
		/> );
		expect( output ).toMatchSnapshot();
	});
});
