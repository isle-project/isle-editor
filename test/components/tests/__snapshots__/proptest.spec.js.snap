// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PropTest /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <label>
    hypothesis-test-for
  </label>
  <p>
    Let p be the population probability of 
    <code>
      Complications
    </code>
     being 
    <code />
    .
  </p>
  <span>
    we-test
  </span>
  <div />
  <label>
    sample-proportion
    : 
    0.055
  </label>
  <pre>
    One-sample z-test

Alternative hypothesis: True mean is not equal to 0.5

    pValue: 0
    statistic: -25.0077
    95% confidence interval: [0.0197,0.0895]


  </pre>
</div>
`;
